File: app/src/main/java/com/example/my_plants/ui/plant/PlantViewModel.java
package com.example.my_plants.ui.plant;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.example.my_plants.Plant;
import com.example.my_plants.R;


public class PlantViewModel extends ViewModel {

    private Plant[] plants = {
            new Plant("MySon", 2, "test")
    };

    private String mText;

    public PlantViewModel() {
        mText = new String();
        mText = plants[0].getName();
    }

    public String getText() {return mText; }
}

File: app/src/main/java/com/example/my_plants/ui/plant/PlantViewModel.kt
package com.example.my_plants.ui.plant

import androidx.lifecycle.ViewModel
import com.example.my_plants.PlantsProvider


class PlantViewModel : ViewModel() {

    private val plants = PlantsProvider.plants

    var text: String? = null
        private set

    init {
        text = String()
        text = "test"//plants[0].name
    }
}